src/deployment/azuredeploy.json

```diff
@@ -865,6 +866,44 @@
                     }
                 ]
             }
+        },
+        {
+            "type": "Microsoft.EventGrid/systemTopics",
+            "apiVersion": "2021-12-01",
+            "name": "[variables('fuzz-blob-topic-name')]",
+            "location": "[resourceGroup().location]",
+            "properties": {
+                "source": "[resourceId('Microsoft.Storage/storageAccounts', variables('storageAccountName'))]",
+                "topicType": "microsoft.storage.storageaccounts"
+            }
+        },
+        {
+            "type": "Microsoft.EventGrid/systemTopics/eventSubscriptions",
+            "apiVersion": "2021-12-01",
+            "name": "[concat(variables('fuzz-blob-topic-name'), '/onefuzz1_subscription')]",
```

Comment:
Why the "1" in the name "/onefuzz1_subscription", was that for debugging purposes?

Reply:
i just followed the existing name. The existing subscription was named `onefuzz1`. I just added the suffix `_subscription`